src/reanimated2/globals.d.ts

```diff
@@ -1 +1,17 @@
 declare const _WORKLET: boolean;
+declare const _stopObservingProgress: (tag: number, flag: boolean) => void;
+declare const _startObservingProgress: (tag: number, flag: boolean) => void;
+declare namespace NodeJS {
+  interface Global {
```

Comment:
Why this is wrapped in Nodejs namespace?

Reply:
https://github.com/software-mansion/react-native-reanimated/blob/d8a02a2a4b5058f267fac9defe1dacd6ed81dd17/src/reanimated2/layoutReanimation/AnimatedRoot.tsx#L29
I found information that `namespace NodeJS` is required for the definition of properties in the global object. But correct me please if I am wrong.